src/modules/ibc.ts

```diff
@@ -131,10 +131,14 @@ export class IBCModule extends BaseModule {
 
       if (chainId && chainInfoMap[chainId]) {
         const cw20RegexArr = denomTrace.baseDenom.match(cw20TokenRegex);
-        const coinGeckoId = tokenClient.geckoTokenNames?.[denom] ?? tokenClient.geckoTokenNames?.[denomTrace.baseDenom] ?? "";
+        const coinGeckoId = tokenClient.geckoTokenNames?.[denom] ?? tokenClient.geckoTokenNames?.[denomTrace.baseDenom];
```

Comment:
Do note that if you remove the default, then in the `getAppCurrency` function, you'll need to declare that `coinGeckoId` param can be `undefined` too.
i.e.
```typescript
getAppCurrency(coinMinimalDenom: string, coinGeckoId: string | undefined, ...): AppCurrency {
  ...
}
```